bundles/binding/org.openhab.binding.lcn/src/main/java/org/openhab/binding/lcn/LcnBindingItem.java

```diff
@@ -0,0 +1,221 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.lcn;
+
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.Set;
+
+import org.openhab.binding.lcn.bindingtarget.*;
+import org.openhab.binding.lcn.common.*;
+import org.openhab.binding.lcn.connection.*;
+import org.openhab.binding.lcn.input.*;
+import org.openhab.core.binding.BindingConfig;
+import org.openhab.core.events.EventPublisher;
+import org.openhab.core.items.Item;
+import org.openhab.core.library.types.DecimalType;
+import org.openhab.core.library.types.StringType;
+import org.openhab.core.types.Command;
+
+/**
+ * Holds all data bound to an openHAB item.
+ * Contains all bindings read from item-configuration.
+ * 
+ * @author Tobias J�ttner
+ */
+class LcnBindingItem implements BindingConfig {
+	
+	/** Single binding. */
+	static class Binding {
+		
+		/** OpenHAB Command associated with the binding. */
+		private final Command cmd;
+		
+		/** The target connection's unique identifier. */
+		private final String connId;
+		
+		/** The binding's target. */
+		private final Target target;
+		
+		/** Flag to indicate this binding has already received status data. */
+		private boolean hasInitialData = false;
+
+		/**
+		 * Constructor for a binding.
+		 * 
+		 * @param connId the target connection's unique identifier
+		 * @param cmd the openHAB command this binding is bound to
+		 * @param lcnTarget the target's configuration
+		 */
+		Binding(Command cmd, String connId, String lcnTarget) {
+			this.cmd = cmd;
+			this.connId = connId;
+			this.target = Target.parse(lcnTarget);
+		}
+		
+		/**
+		 * Gets the openHAB command the binding is bound to.
+		 * Might be an empty dummy for "visualization-bindings".
+		 * 
+		 * @return the openHAB command
+		 */
+		Command getOpenHabCmd() {
+			return this.cmd;
+		}
+		
+		/**
+		 * Gets the target's connection identifier.
+		 * 
+		 * @return the connection identifier
+		 */
+		String getConnId() {
+			return this.connId;
+		}
+		
+		/**
+		 * Get the parsed target.
+		 * 
+		 * @return the target
+		 */
+		Target getTarget() {
+			return this.target;
+		}
+		
+		/**
+		 * Gets the flag that indicates if this binding has already received status data.
+		 * 
+		 * @return true if the binding has already received status data
+		 */
+		boolean hasInitialData() {
+			return this.hasInitialData;
+		}
+		
+		/**
+		 * Sets the flag that indicates if this binding has already received status data.
+		 * 
+		 * @param hasInitialData the flag
+		 */
+		void setInitialData(boolean hasInitialData) {
+			this.hasInitialData = hasInitialData;
+		}
+		
+	}
+	
+	/** The actual underlying openHAB item. */
+	private final Item item;
```

Comment:
any local references to Items should be avoided. It seems in this case you don't really need the Item at all (didn't find any usage of the item in subsequent method calls.